src/utilities/rgb565torgb.js

```diff
@@ -0,0 +1,11 @@
+export function rgb565torgb( rgb565 ) {
+
+	let r = ( rgb565 & 0xF800 ) >> 11;
+	let g = ( rgb565 & 0x07E0 ) >> 5;
+	let b = rgb565 & 0x001F;
+	r = ( r << 3 ) | ( r >> 2 ); // Scale up red component
+	g = ( g << 2 ) | ( g >> 4 ); // Scale up green component
+	b = ( b << 3 ) | ( b >> 2 ); // Scale up blue component
```

Comment:
These all seem like magic numbers. Does this 565 unpacking implementation come from somewhere?

Reply:
Hey, I updated the 565 unpacking to a much cleaner one I found over here [link](https://rgbcolorpicker.com/565), the previous one was taken from a arduino forum.

Reply:
Great, thanks! Can we add a comment with a link to that page - then I'll test it and get this merged!

Reply:
ok 👍